src/Toggle/Toggle.js

```diff
@@ -120,6 +120,10 @@ class Toggle extends Component {
      */
     thumbStyle: PropTypes.object,
     /**
+    * Override the inline styles for thumb when the toggle switch is toggled on.
+    */
+    thumbWhenSwitchedStyle: PropTypes.object,
```

Comment:
Sorry -- I misread the source for a moment. I tripped up on the fact that we have styles object props named that way in `getStyles()` (which is where @agoldis got the naming convention from) and thought it was another prop for a moment.

```js
    trackWhenSwitched: {
      backgroundColor: toggle.trackOnColor,
    },
    thumbWhenSwitched: {
      backgroundColor: toggle.thumbOnColor,
      left: '100%',
    },
    trackWhenDisabled: {
      backgroundColor: toggle.trackDisabledColor,
    },
    thumbWhenDisabled: {
      backgroundColor: toggle.thumbDisabledColor,
    },
```